uaclient/entitlements/repo.py

```diff
@@ -77,6 +77,17 @@ def packages(self) -> "List[str]":
     def repo_key_file(self) -> str:
         pass
 
+    @property
+    def dpkg_options(self) -> "List[str]":
+        """
+        Returns a list of dpkg options to pass to apt
+        """
+        options_list = ['force-confdef', 'force-confold']
+        dpkg_options = []
+        for option in options_list:
+            dpkg_options.append('-o="Dpkg::Options::=--{}"'.format(option))
+        return dpkg_options
```

Comment:
This looks like the right solution thank you much @jaredledvina for proposing it.

Can we adapt the method here a bit for readability and intent. What do you think of the following?
```suggestion
    @property
    def dpkg_options(self) -> "List[str]":
        """
        Return a list of dpkg options to avoid interactive prompts

        Automatically preserve local configuration files if changes detected.
        https://debian-handbook.info/browse/stable/sect.package-meta-information.html#sidebar.questions-conffiles
        """
        return [
            '-o="Dpkg::Options::=--force-confdef"',
            '-o="Dpkg::Options::=--force-confold"',
        ]
```